src/components/LineSeries.tsx

```diff
@@ -29,16 +30,16 @@ export function LineSeries(props: LineSeriesProps) {
     ...otherProps
   } = props;
   const lineStyle = functionalStyle({}, OldLineStyle, { id });
+  const isVisible = useIsSeriesVisible(id);
   useEffect(() => {
```

Comment:
@targos you can see here
every moment we're adding and removing legend label
(it's not working now)

Reply:
we're changing LegendLabels state